packages/site/src/components/App.js

```diff
@@ -17,17 +17,19 @@
 /* @flow */
 import React from 'react';
 import { Redirect, Route, Switch } from 'react-router-dom';
-import {
-  createStyledComponent,
-  ThemeProvider
-} from '@mineral-ui/component-utils';
+import { createStyledComponent } from '@mineral-ui/component-utils';
 import createKeyMap from '../utils/createKeyMap';
 import ComponentDoc from './ComponentDoc';
 import Footer from './Footer';
 import _Nav from './Nav';
-import siteTheme from './siteTheme';
 import styleReset from './styleReset';
 
+import GettingStarted from '../pages/GettingStarted';
```

Comment:
If you are going to always be importing all of the pages, you could add a pages/index, and just import that here.